### PR TITLE
Cu 86b1e6xf3 fix the connected page showing whenever the air pods are connected

### PR DESCRIPTION
**Pull request by whamester**
Note: I removed the lorem ipsum text for this week's demo

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b1e6xf3 Fix the `Connected` page showing whenever the AirPods are connected](https://app.clickup.com/t/86b1e6xf3)

File: src/app/setup/earbuds-connected.tsx [hunk 1]
```diff
@@ -99,10 +99,6 @@ const EarbudsTrainingScreen = () => {
                 >
                   You’re connected!
                 </Text>
-                <Text align="center">
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
```
Comment (ReinhardtBotha) [inline src/app/setup/earbuds-connected.tsx:103]: are we not going to add any text here in place of the lorem ipsum?

**Review by ReinhardtBotha — APPROVED** [review]
✅

**Review by Victorportus — APPROVED** [review]

**Review by zeckkimon — APPROVED** [review]

File: src/app/setup/earbuds-connected.tsx (hunk 1 above)
Comment (whamester) [reply src/app/setup/earbuds-connected.tsx:103]: as we are going to re create these pages, I just removed the lorem ipsum for this friday's demo. The new onboarding won't be shown this friday.

**Pull request merged**